src/Components/Assets/AssetType/ONVIFCamera.tsx

```diff
@@ -111,10 +116,12 @@ const ONVIFCamera = (props: ONVIFCameraProps) => {
           )
         )
       );
+      setLoadingAddPreset(false);
       if (res?.status === 201) {
         Notification.Success({
           msg: "Preset Added Successfully",
         });
+        setBed({});
```

Comment:
@Pranshu1902 why are we resetting the bed here?

Reply:
@khavinshankar So that after adding a Preset, the `Bed` input field of the form is cleared just like the `Preset Name` field.
![image](https://user-images.githubusercontent.com/70687348/216253317-626e8419-76c2-4499-9c40-06a7ff85d245.png)